src/packages/BooleanQueryTokenizer/BooleanQueryTokenizer.ts

```diff
@@ -148,6 +155,94 @@ export class BooleanQueryTokenizer {
         );
     }
 
+    public static extractWords(inputString: string): Extract {
```

Comment:
I think we need to add doc here or on result type to describe that we are going to exclude negationed words and phrases